### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by leonjia0112**
Add dependency requirements to the README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,13 @@ TPM emulator is a program that runs in the deamon to mimic TPM commands.
 
 ## Prerequisites
 
+### Required Package
```
Comment (lukehinds) [inline README.md]: Required Package**s**

File: README.md [hunk 2]
```diff
@@ -25,6 +25,13 @@ TPM emulator is a program that runs in the deamon to mimic TPM commands.
...
+Rust-keylime agent rust code implementation require the following packages to both compile time and run time.
```
Comment (lukehinds) [inline README.md]: This rolls off the tongue a little easier..

> The rust-keylime agent requires the following packages for both compile and run time.

Comment (frozencemetery) [inline README.md]: Please note that these are *Fedora* packages.  (If you want to include the Debian names, that would also be fine.)  Formatting this as a `dnf install ...` would also be helpful.

**Comment by frozencemetery** [comment]
The url subject of your commit message should capture what the change is, not just what files have changed.  For a change this small, there shouldn't be a body.

**Comment by leonjia0112** [comment]
Request change resolved. Ready for a review.
@frozencemetery @lukehinds @mbestavros

**Comment by leonjia0112** [comment]
> I would change the commit message as `Update README.md with package dependencies`. This way when we do releases, we can use `git log` and grab the brief details of the change for the release notes.

This one sounds better! Thanks!

**Pull request merged**